internal/suggestions/suggest.go

```diff
@@ -184,11 +184,11 @@ func Suggest(schema []byte, schemaPath string, errs []error, config Config) (*Sc
 
 		suggest.Verbose = false
 		continueSuggest := true
-		numSuggestions := min(suggestionBatchSize, len(errs))
 		var err error
 
 		// Request suggestions in parallel, in batches of at most suggestionBatchSize
 		for continueSuggest {
+			numSuggestions := min(suggestionBatchSize, len(errs))
```

Comment:
fixes bug, need to update `errs` on each application so we don't get out of bounds exception